docs/book.toml

```diff
@@ -8,6 +8,12 @@ title = "bootc"
 [preprocessor.mermaid]
 command = "mdbook-mermaid"
 
+[preprocessor.header-footer]
+headers = []
+footers = [
+  { padding = "\n---\n<p style=\"font-size:11px\">The Linux Foundation® (TLF) has registered trademarks and uses trademarks. For a list of TLF trademarks, see <a href=\"https://www.linuxfoundation.org/trademark-usage/\">Trademark Usage.</a></p>" }
```

Comment:
This HTML-in-TOML is a bit syntactically awkward, but I guess there's nothing we can do about that.